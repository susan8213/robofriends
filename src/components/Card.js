import React from 'react';
import "tachyons";

const Card = ({ id, name, email }) => {
    return (
        <div className="br3 pa3 ma2 tc bg-lightest-blue bw2 ba b--black-10 dib grow">
            <img 
                src={`https://robohash.org/${id}?set=set4&size=250x250`}
                alt="Cat" 
                className="br-100 h4 w4 dib pa2"
            ></img>
            <h1 className="f3 mb2">{name}</h1>
            <h2 className="f5 fw4 dark-gray mt0">{email}</h2>
        </div>
    )
}

export default Card;