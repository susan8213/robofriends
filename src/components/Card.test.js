import { shallow } from 'enzyme';
import React from 'react';
import Card from './Card';


describe('<Card />', () =>{
    it('smoke test', () => {
        expect([]).toEqual([]);
    });

    it('render without crashing', () => {
        expect(shallow(<Card/>).html()).toMatchSnapshot();
    })

    it('contains img tag', () => {
        const wrapper = shallow(<Card/>);
        expect(wrapper.find('img')).toHaveLength(1);
    })
})