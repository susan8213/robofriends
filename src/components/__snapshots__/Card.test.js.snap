// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> render without crashing 1`] = `"<div class=\\"br3 pa3 ma2 tc bg-lightest-blue bw2 ba b--black-10 dib grow\\"><img src=\\"https://robohash.org/undefined?set=set4&amp;size=250x250\\" alt=\\"Cat\\" class=\\"br-100 h4 w4 dib pa2\\"/><h1 class=\\"f3 mb2\\"></h1><h2 class=\\"f5 fw4 dark-gray mt0\\"></h2></div>"`;
